Sources/Controllers/Cells/OASegmentTableViewCell.m

```diff
@@ -20,4 +20,22 @@ - (void)setSelected:(BOOL)selected animated:(BOOL)animated
     [super setSelected:selected animated:animated];
 }
 
+- (void)layoutSubviews
+{
+    [super layoutSubviews];
+    
+    for (UIView *view in self.subviews)
+    {
+        if ([NSStringFromClass(view.class) hasSuffix:@"CellSeparatorView"])
```

Comment:
What does 'CellSeparatorView' mean? Please refrain from using private APIs. There's no guarantee that in future versions of iOS, Apple will use the same name